cli/src/tasks/new-plugin.ts

```diff
@@ -221,47 +221,61 @@ function generateAndroidManifest(domain: string, pluginPath: string) {
 
 function generatePackageJSON(answers: NewPluginAnswers, cliVersion: string) {
   return {
-    name: answers.name,
-    version: '0.0.1',
-    description: answers.description,
-    main: 'dist/esm/index.js',
-    types: 'dist/esm/index.d.ts',
-    scripts: {
-      'build': 'npm run clean && tsc',
+    'name': answers.name,
+    'version': '0.0.1',
+    'description': answers.description,
+    'main': 'dist/plugin.js',
+    'module': 'dist/esm/index.js',
+    'types': 'dist/esm/index.d.ts',
+    'scripts': {
+      'lint': 'npm run prettier -- --check && npm run swiftlint -- lint',
+      'prettier': 'prettier "**/*.{css,html,ts,js,java}"',
+      'swiftlint': 'swiftlint',
+      'build': 'npm run clean && tsc && rollup -c rollup.config.js',
       'clean': 'rimraf ./dist',
       'watch': 'tsc --watch',
       'prepublishOnly': 'npm run build'
     },
-    author: answers.author,
-    license: answers.license,
-    dependencies: {
-      '@capacitor/core': `^${cliVersion}`
-    },
-    devDependencies: {
-      'rimraf': '^3.0.0',
-      'typescript': '^3.2.4',
+    'author': answers.author,
+    'license': answers.license,
+    'devDependencies': {
+      '@capacitor/android': `^${cliVersion}`,
+      '@capacitor/core': `^${cliVersion}`,
       '@capacitor/ios': `^${cliVersion}`,
-      '@capacitor/android': `^${cliVersion}`
+      '@ionic/prettier-config': '^1.0.0',
+      '@ionic/swiftlint-config': '^1.0.0',
+      '@rollup/plugin-node-resolve': '^8.1.0',
+      'prettier': '^2.0.5',
+      'prettier-plugin-java': '^0.8.0',
+      'rimraf': '^3.0.0',
+      'rollup': '^2.21.0',
+      'swiftlint': '^0.39.4',
+      'typescript': '~3.8.3'
+    },
+    'peerDependencies': {
+      '@capacitor/core': `^${cliVersion}`
     },
-    files: [
+    'files': [
       'dist/',
       'ios/',
       'android/',
       `${fixName(answers.name)}.podspec`
     ],
-    keywords: [
+    'keywords': [
       'capacitor',
       'plugin',
       'native'
     ],
-    capacitor: {
-      ios: {
+    'capacitor': {
+      'ios': {
         src: 'ios',
```

Comment:
since you are wrapping everything with `''`, shouldn't src be wrapped too? 
```suggestion
        'src': 'ios',
```